nntrainer/layers/flatten_layer.cpp

```diff
@@ -49,6 +71,7 @@ void FlattenLayer::setProperty(const std::vector<std::string> &values) {
 
 void FlattenLayer::exportTo(Exporter &exporter,
                             const ml::train::ExportMethods &method) const {
+  exporter.saveResult(flatten_props, method, this);
```

Comment:
When exporting Tflite, since graphs and tensors are changed for export, it seems that if you call saveResult twice, it will not work properly.

Reply:
I will also look for a way to solve this problem.

Reply:
Removed this line